### PR TITLE
fixes if statement

### PR DESCRIPTION
**Pull request by jtroussard**
I believe you meant to have a comparator here instead of a assignment.

Please review and accept! Thanks.

### CODE REVIEW

File: motor_test/motor_test.ino [hunk 1]
```diff
@@ -35,7 +35,8 @@ void loop()
     Serial.print("stopped, ");
     break;
   }
-  if (running = true)
+  // This should make th motor wait for the serial command
```
Comment (Sp00kyBones) [inline motor_test/motor_test.ino]: //please spell the correctly

**Review by Sp00kyBones — CHANGES_REQUESTED** [review]

File: motor_test/motor_test.ino (hunk 1 above)
Comment (jtroussard) [reply motor_test/motor_test.ino]: 👌 yes master

**Comment by Sp00kyBones** [comment]
please finish my project for me thanks!!!!

**Comment by jtroussard** [comment]
i could write something

**Review by Sp00kyBones — APPROVED** [review]

**Comment by Sp00kyBones** [comment]
cool thanks!

**Pull request merged**